### PR TITLE
[plugin-rest-api] Dont perform JSON nested steps if number of element…

### PR DESCRIPTION
**Pull request by uarlouski**
…s is 0 and rule is matched

### CODE REVIEW

File: vividus-tests/src/main/resources/story/integration/JsonResponseValidationSteps.story [hunk 1]
```diff
@@ -57,6 +57,11 @@ When I find greater than `1` JSON elements by `$.store.book` and for each elemen
 |step                                                       |
 |Then number of JSON elements by JSON path `$.author` is = 1|
 
+Scenario: Verify step 'When I find $comparisonRule `$elementsNumber` JSON elements from `$json` by `$jsonPath` and for each element do$stepsToExecute' with zero elements
+When I find <= `1` JSON elements from `{}` by `$.name` and for each element do
+|step                    |
+|Then `1` is equal to `1`|
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/JsonResponseValidationSteps.story:63]: let's put here some step, that should fail, like:
```
Then `0` is equal to `1`
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by abudevich — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Comment by ikalinin1** [comment]
@uarlouski  could you please update the docs?

**Comment by uarlouski** [comment]
> @uarlouski could you please update the docs?

@ikalinin1 isn't it behavior expected by end-users? :)

**Comment by ikalinin1** [comment]
> > @uarlouski could you please update the docs?
> 
> @ikalinin1 isn't it behavior expected by end-users? :)

Depends on the user -> worth mentioning in the docs :)

**Review by valfirst — APPROVED** [review]

**Pull request merged**